### ISSUE TITLE
Multilingual support?

### ISSUE THREAD

**Issue by ourstudio-erikblomqvist** [zurb/panini#180]
Hello,

After reading through [issue #45](https://github.com/zurb/panini/issues/45), I thought that my request would have been implemented, but looking through the source code of Panini, I couldn't find any evidence of "i18n" being implemented.

Could somebody guide me on how one could add a YML/JSON file to add multilingual content to partials within a project?

Thanks

**Comment by elcaptain** [comment]
Hey @ourstudio-erikblomqvist 

I started using [gulp-html-i18n](https://github.com/webyom/gulp-html-i18n) for my translations. It's super easy to integrate (with foundation for emails at least). Maybe it fits your needs too.

**Comment by DanielRuf** [comment]
Hi @elcaptain,

this looks interesting. Do you want to provide a PR to add this to the docs?

**Comment by elcaptain** [comment]
Hey @DanielRuf, 
Do you mean the readme.md or which docs are you referring to? What would be a good way to do that? Adding a section like "useful libraries"?

**Comment by DanielRuf** [comment]
Yes, probably. As this would be helpful for others too (to know how they can achieve this).
